docs/self-managed/optimize-deployment/configuration/service-config.yaml

```diff
@@ -440,30 +439,30 @@ historyCleanup:
 locales:
   # all locales available
   # Note: for others than the default there must be a <localeCode>.json file available under ./config/localization.
-  availableLocales: ['en', 'de']
+  availableLocales: ["en", "de"]
   # the fallback locale is used if there is a locale requested that is not available in availableLocales
-  fallbackLocale: 'en'
+  fallbackLocale: "en"
 
 ui:
   header:
     # determines the color theme of the text in the header. Currently 'dark' and 'light' are supported.
-    textColor: 'dark'
+    textColor: "dark"
     # Path to the logo that is displayed in the header of Optimize.
     # Path can be:
     # * relative: starting from the config folder you can provide a relative path.
     # * absolute: full path in the file system.
     #
     # Supported image formats can be found here:
     # https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Supported_image_formats
-    pathToLogoIcon: 'logo/camunda_icon.svg'
+    pathToLogoIcon: "logo/camunda_icon.svg"
     # a hex encoded color that should be used as background color for the header. Default color is white.
-    backgroundColor: '#FFFFFF'
+    backgroundColor: "#FFFFFF"
 
 eventBasedProcess:
   # A list of userIds that are authorized to manage (Create, Update, Publish & Delete) event based processes.
   authorizedUserIds: []
-    # A list of groupIds that are authorized to manage (Create, Update, Publish & Delete) event based processes.
-    authorizedGroupIds: []
+  # A list of groupIds that are authorized to manage (Create, Update, Publish & Delete) event based processes.
+  authorizedGroupIds: []
```

Comment:
I had to outdent this block in order to commit -- yaml didn't like it being indented deeper than `authorizedUserIds` above it. 

I assumed this was the intention -- if you know these settings and know that I was incorrect, let me know!

Reply:
Good question for @menski

Reply:
I think this was unintended and the fix is correct if I look at the default [config file.](https://github.com/camunda/camunda-optimize/blob/67de788cb2263628e7247305bf7d3261c787dcbe/distro/src/config/environment-config.yaml#L291-L294)